index.js

```diff
@@ -190,7 +190,10 @@ class ExecutorQueue extends Executor {
             ...args
         };
 
-        logger.info(`${options.method} ${options.path} for pipeline ${config.pipelineId}:${config.jobId}`);
+        logger.info(
+            `${options.method} ${options.path} for pipeline ${config.pipelineId || config.pipeline.id}:${config.jobId ||
```

Comment:
is there any path here where pipelineId and pipeline is empty ? that will cause error .
 For example with this change `stats()` might error out.